src/fileseq/filesequence.py

```diff
@@ -434,3 +434,14 @@ def getPaddingChars(num):
             return "#" * (num / 4)
         else:
             return "@" * num
+
+    @staticmethod
+    def getPaddingNum(chars):
+        """
+        Given a supported group of padding characters, return the amount of padding.
+        :param chars: a supported group of padding characters (str)
+        :return: int
+        """
+        mapping = {'@': 1, '#': 4}
+
+        return sum([mapping[char] for char in chars])
```

Comment:
Yep. That would be preferable. Personally, I would expect the method to raise if unsupported characters were passed in rather than just ignore them. 

Just noticed that I should also be using PAD_MAP from the constants module.